### ISSUE TITLE
[Acceptance] [InnerBrowser] Sending unchecked checkboxes

### ISSUE THREAD

**Issue by albertboada** [Codeception/Codeception#2275]
As of now, the following doesn't work in Codeception acceptance tests:

In browsers, if you submit a form with these fields:

``` html
<input type="hidden" name="terms" value="No">
<input type="checkbox" name="terms" value="Yes">
```
- If the checkbox is **not checked**, the **hidden field** will be sent.
- If the checkbox **is** checked, this one will be sent, since it's the last occurrence of the field named `terms`.

Codeception is not reproducing this browser default behaviour. The `terms` param is not being included in the request when the checkbox is unchecked.

This is a pretty common practice amongst frameworks like CakePHP.

**Comment by albertboada** [comment]
Ok, sadly enough, and just for reference, I have just seen this is basically Symfony's DomCrawler's fault. It's proposed to be fixed in Symfony 3.0 (https://github.com/symfony/symfony/issues/11742)

>
> DomCrawler
> Add support for fallback hidden fields. (ref)
>

**Comment by githubjeka** [comment]
Functional tests have same the problem.

Constantly have to follow:

```php
$I->submitForm("#edit-form",
     [              
         'ModelForm[off]' => 0,
     ]
);
```

**Comment by eXorus** [comment]
Should be fixed in SF 3.0 I hope...
I'm closing this issue, feel free to reopen if it is not resolved.
